pages/api/sendgrid/addContact.js

```diff
@@ -0,0 +1,39 @@
+var request = require("request");
```

Comment:
you are using the node built-in package `fetch` and `request` is doing the same, I will remove in the future, so we keep the bundle small, but OK for now